base_import_match/__manifest__.py

```diff
@@ -1,14 +1,13 @@
-# -*- coding: utf-8 -*-
 # Copyright 2016 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
 # Copyright 2016 Tecnativa - Vicent Cubells
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Base Import Match",
     "summary": "Try to avoid duplicates before importing",
-    "version": "10.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Tools",
     "website": "https://tecnativa.com",
-    "author": "Grupo ESOC Ingeniería de Servicios,"
+    "author": "Grupo ESOC Ingeniería de Servicios, "
```

Comment:
You can remove this one, as it's the old Jairo's company, and the development has been absorbed by us.

Reply:
Done